packages/blockchain-wallet-v4-frontend/src/components/Faq/template.js

```diff
@@ -40,14 +40,14 @@ const Faq = (props) => {
 
   return (
     <Fragment>
-      <ModalHeader onClose={() => handleTrayRightToggle()}>
-        <FormattedMessage id='layouts.wallet.trayright.faq' defaultMessage='Frequently Asked Questions'/>
+      <ModalHeader onClose={() => handleTrayRightToggle()} >
```

Comment:
Small thing and doesn't matter but the space is only necessary before the closing tag.